releasenotes/notes/unary-arithmetic-7db455017e1faa45.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features_circuits:
```

Comment:
you should also mention here the synthesis functions (Unary Arithmetic Synthesis) and the HLS plugins (Integer comparators, Sums).